libs/domains/services/feature/src/lib/service-avatar/service-avatar.tsx

```diff
@@ -25,9 +25,16 @@ export const ServiceAvatar = forwardRef<ElementRef<typeof Avatar>, ServiceAvatar
   return (
     <Avatar
       ref={ref}
+      className={className}
```

Comment:
No need to add this as remaining `props` are already spread line 42

Reply:
Yes my bad https://github.com/Qovery/console/pull/1630/commits/e592c870cdf2af9eb23054ca396732f4f16a36f5